HPXMLtoOpenStudio/resources/geometry.rb

```diff
@@ -569,7 +569,7 @@ def self.apply_occupants(model, runner, hpxml_bldg, num_occ, space, schedules_fi
   end
 
   def self.get_occupancy_default_num(nbeds)
-    return Float(nbeds)
+    return [Float(nbeds), 1.0].max
```

Comment:
Without this change, a 0 bedroom unit that does not have the number of residents set (i.e., an asset calculation) would be "vacant".